public/directives/wz-logtest/components/logtest.tsx

```diff
@@ -80,22 +79,34 @@ export const Logtest = compose(
       `nist_800_53: ${JSON.stringify((result.rule || '').nist_800_53 || '-')} \n    ` +
       `pci_dss: ${JSON.stringify((result.rule || '').pci_dss || '-')} \n    ` +
       `tsc: ${JSON.stringify((result.rule || '').tsc || '-')} \n` +
-      `**Alert to be generated.`
+      `**Alert to be generated. \n\n\n`
     );
   };
 
-  const test = async () => {
+  const runAllTest = () => {
+    setTestResult('');
+    value.map((event) => {
+      test(event);
+    });
```

Comment:
As a suggestion, you could have used `.forEach` array method because `.map` is more expensive due to creating a new array as result.